tools/building.py

```diff
@@ -560,6 +560,10 @@ def link_lld(args, target, external_symbol_list=None):
   for a in llvm_backend_args():
     cmd += ['-mllvm', a]
 
+  # LLVM has turned on the new pass manager by default, but it causes some code
+  # size regressions. For now, use the legacy one.
```

Comment:
Can you link to bug here?  (either on our side of in llvm)

Reply:
(feel free to using admin privs to land this after adding that.. we don't want to wait for tests again)